.github/workflows/pocket-ic-build-windows.yml

```diff
@@ -0,0 +1,26 @@
+name: 'PocketIC library build on Windows'
+on:
+  pull_request:
+    paths:
+      - packages/pocket-ic/**
+  push:
+    branches:
+      - master
+# runs for the same workflow are cancelled on PRs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
```

Comment:
This needs to be updated because it will fail on master. The two options are:

Cancel in progress jobs on both PRs and on master:
```
group: ${{ github.workflow }}-${{ github.ref }}
```
Cancel in progress jobs on PRs but run on every push to master:
 ```
group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```